lib/beaker-pe/install/pe_utils.rb

```diff
@@ -578,6 +581,39 @@ def simple_monolithic_install(master, agents, opts={})
           end
         end
 
+
+        # Configure the master to use a proxy and drop unproxied connections
+        def config_master_for_proxy_access
+          step "configuring master to use proxy" do
+            @osmirror_host = "osmirror.delivery.puppetlabs.net"
+            @osmirror_host_ip = IPSocket.getaddress(@osmirror_host)
+            @delivery_host = "enterprise.delivery.puppetlabs.net"
+            @delivery_host_ip = IPSocket.getaddress(@delivery_host)
+            @proxy_ip = @options[:proxy_ip]
+            @proxy_hostname = @options[:proxy_hostname]
+            @master_ip = on master, "hostname -I | tr '\n' ' '"
+            on master, "echo \"#{@proxy_ip}  #{@proxy_hostname}\" >> /etc/hosts"
+            on master, "echo \"#{@master_ip.stdout}  #{master.connection.vmhostname}\" >> /etc/hosts"
+            on master, "echo \"#{@osmirror_host_ip}    #{@osmirror_host}\" >> /etc/hosts"
+            on master, "echo \"#{@delivery_host_ip}    #{@delivery_host}\" >> /etc/hosts"
+            on master, "iptables -A OUTPUT -p tcp -d #{master.connection.vmhostname} -j ACCEPT"
+            on master, "iptables -A OUTPUT -p tcp -d 10.32.0.0/16 -j ACCEPT"
```

Comment:
This is the internal Puppet LAN, right? Could we make a comment to that effect?

Reply:
Yeah, good call.  Added that comment.

Reply:
👍 thanks